Platform/AlderlakeBoardPkg/Library/Stage1BBoardInitLib/Stage1BBoardInitLib.c

```diff
@@ -372,26 +373,34 @@ TpmInitialize (
   BootMode     = GetBootMode();
   PlatformData = (PLATFORM_DATA *)GetPlatformDataPtr ();
 
-  if((PlatformData != NULL) && PlatformData->BtGuardInfo.MeasuredBoot &&
-    (!PlatformData->BtGuardInfo.DisconnectAllTpms) &&
-    ((PlatformData->BtGuardInfo.TpmType == dTpm20) || (PlatformData->BtGuardInfo.TpmType == Ptt))){
 
-    //  As per PC Client spec, SRTM should perform a host platform reset
-    if (PlatformData->BtGuardInfo.TpmStartupFailureOnS3) {
-      ResetSystem(EfiResetCold);
-      CpuDeadLoop ();
+  if ((PlatformData != NULL) && MEASURED_BOOT_ENABLED() &&
+    (!PlatformData->BtGuardInfo.DisconnectAllTpms)) {
+
+    if ((PlatformData->BtGuardInfo.MeasuredBoot == TRUE) &&
```

Comment:
" == TRUE" could be removed since BOOLEAN doesn't need compare.

Reply:
done